rust/bridge/jni/src/lib.rs

```diff
@@ -4,6 +4,7 @@
 //
 
 #![allow(clippy::missing_safety_doc)]
+#![deny(clippy::unwrap_used)]
```

Comment:
This needs to get added to bridge/ffi/src/lib.rs and bridge/node/src/lib.rs as well.